app/serializers/user_serializer.rb

```diff
@@ -13,109 +13,12 @@ def id
     nil
   end
 
-  def account
```

Comment:
This cleanup of the user serializer is great 👍 Most serializers should be simple like this with no control flow. If we clean up all the serializers we may be able to get a performance improvement in the future by swapping out active model serializer for [fast_jsonapi](https://github.com/Netflix/fast_jsonapi ).

Reply:
:100: